utils/logger.go

```diff
@@ -10,7 +11,7 @@ func NewFilePathDebugLogger(filepath, name string) (logging.Logger, error) {
 	logConfig := logging.NewLoggerConfig()
 	logConfig.OutputPaths = append(logConfig.OutputPaths, filepath)
 	logConfig.ErrorOutputPaths = append(logConfig.ErrorOutputPaths, filepath)
-
+	logConfig.Level.SetLevel(zapcore.DebugLevel)
```

Comment:
This fixes the log level not being set to debug level in the motion planning & nav file path logger.